core/controllers/voice_artist_test.py

```diff
@@ -12,293 +14,118 @@
-        self.assertEqual(
-            exp_user_data.draft_change_list, self.VALID_DRAFT_CHANGELIST)
-        self.assertEqual(exp_user_data.draft_change_list_id, 1)
-        self.assertEqual(
-            response, {'status_code': 400,
-                       'error': (
-                           'Voice artist does not have permission to make'
-                           ' some changes in the change list.')
-                      })
-
-    def test_draft_updated_version_invalid(self):
-        payload = {
-            'change_list': self.VALID_DRAFT_CHANGELIST,
-            'version': 10,
-        }
-        response = self.put_json(
-            '/createhandler/autosave_draft/%s' % self.EXP_ID,
-            payload, csrf_token=self.csrf_token)
-        exp_user_data = user_models.ExplorationUserDataModel.get_by_id(
-            '%s.%s' % (self.voice_artist_id, self.EXP_ID))
-        self.assertEqual(
-            exp_user_data.draft_change_list, self.VALID_DRAFT_CHANGELIST)
-        self.assertEqual(exp_user_data.draft_change_list_exp_version, 10)
-        self.assertFalse(response['is_version_of_draft_valid'])
-        self.assertEqual(response['draft_change_list_id'], 2)
-
-    def test_discard_draft(self):
-        self.post_json(
-            '/createhandler/autosave_draft/%s' % self.EXP_ID, {},
-            csrf_token=self.csrf_token)
-        exp_user_data = user_models.ExplorationUserDataModel.get_by_id(
-            '%s.%s' % (self.voice_artist_id, self.EXP_ID))
-        self.assertIsNone(exp_user_data.draft_change_list)
-        self.assertIsNone(exp_user_data.draft_change_list_last_updated)
-        self.assertIsNone(exp_user_data.draft_change_list_exp_version)
-
-
-class TranslationFirstTimeTutorialTest(BaseVoiceArtistControllerTests):
-    """This controller tests the first time tutorial for translations."""
-
-    EXP_ID = 'exp1'
-
-    def setUp(self):
-        super(TranslationFirstTimeTutorialTest, self).setUp()
-        self.login(self.OWNER_EMAIL)
-        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
-        self.save_new_valid_exploration(self.EXP_ID, self.owner_id)
-        rights_manager.assign_role_for_exploration(
-            self.owner,
-            self.EXP_ID,
-            self.voice_artist_id,
-            rights_domain.ROLE_VOICE_ARTIST
-        )
-        self.logout()
-
-        self.login(self.VOICE_ARTIST_EMAIL)
-        # Generate CSRF token.
-        self.csrf_token = self.get_new_csrf_token()
-
-    def test_firsttime_translation_tutorial(self):
-        """Testing of the firsttime translation tutorial http requests."""
-        # Check if method returns 200 http status.
-        self.post_json(
-            '/createhandler/started_translation_tutorial_event/%s'
-            % self.EXP_ID, {}, csrf_token=self.csrf_token,
-            expected_status_int=200)
+import python_utils
+
+import mutagen
+from mutagen import mp3
+
+
+class AudioUploadHandler(base.BaseHandler):
```

Comment:
Is this an expected change? (In general, go through the CL on GitHub after creating PR or pushing new commits.

Reply:
Mistake. I have changed to correct one.